client/doc/onboarding.md

```diff
@@ -0,0 +1,64 @@
+## Onboarding component
```

Comment:
@gregorylegarec : Sorry but I d'ont see the point of this file; it seems to me that it is overide `doc/api`.